### DISCUSSION TITLE
ERROR: initializing node: starting node: could not build arguments for function...

### DISCUSSION THREAD

**Discussion by Sangmin1514** [filecoin-project/lotus#9498] [General Q&A] [answered]
https://lotus.filecoin.io/tutorials/lotus-miner/run-a-miner/

Running on Lotus daemon on Miner setup-Configuration give the following error:

ERROR: initializing node: starting node: could not build arguments for function...

Why does it happen? Please how to solve it.

**Comment by rjan90** [comment, marked as answer]
Hey @Sangmin1514! Can you paste the full output of the error? It will help determine what the cause is 😄 

_Edit:_
These types of error usually comes from a not properly cleaned repo / run a build for a different network / etc.

Deleting the repo and re-cloning, or building with a different LOTUS_PATH=/ should fix it :)

**Reply by Sangmin1514** [reply]
ERROR: initializing node: starting node: could not build arguments for function "github.com/filecoin-project/lotus/node/modules/lp2p".StartListening.func1 (/home/circleci/project/node/modules/lp2p/addrs.go:85): failed to build host. Host: could not build arguments for function "reflect".makeFuncStub (/usr/local/go/scr/reflect/asm_amd64.s:30): failed to build lp2p.BaseIpfsRouting: could not build arguments for function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s30):failed to dtype.NetworkName: could not build arguments for funcion "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s30): failed to dtypes.AfterGenesisSet: received non-nil error from "reflect.makeFuncStub (/usr/local/go.src/reflect/asm_amd64.s:30) getting genesis block failed: ipld: could not find bafy2bza................dmm2

**Reply by rjan90** [reply]
Thanks! These types of error usually comes from a not properly cleaned repo / run a build for a different network / etc.

Deleting the repo and re-cloning, or building with a different LOTUS_PATH=/ should fix it :)

**Reply by Sangmin1514** [reply]
Thank you! :)
